blocks/init/src/blocks/components/paragraph/paragraph.php

```diff
@@ -14,31 +14,24 @@
   return;
 }
 
-$content      = $paragraph['content'] ?? '';
-$remove_style = $paragraph['removeStyle'] ?? false;
-
-if ( ! $remove_style ) {
-  $component_class = 'paragraph';
-  $block_class     = $attributes['blockClass'] ?? '';
-  $style_align     = isset( $paragraph['styleAlign'] ) ? "{$component_class}__align--{$paragraph['styleAlign']}" : '';
-  $style_color     = isset( $paragraph['styleColor'] ) ? "{$component_class}__color--{$paragraph['styleColor']}" : '';
-  $style_size      = isset( $paragraph['styleSize'] ) ? "{$component_class}__size--{$paragraph['styleSize']}" : '';
-
-  $paragraph_class = "
-    {$component_class}
-    {$style_color}
-    {$style_align}
-    {$style_size}
-    {$block_class}__paragraph
-  ";
-
-  ?>
-
-  <p class="<?php echo esc_attr( $paragraph_class ); ?>">
-    <?php echo wp_kses_post( $content ); ?>
-  </p>
-
-  <?php
-} else {
-  echo wp_kses_post( $content );
-}
+$content = $paragraph['content'] ?? '';
+
+$component_class = 'paragraph';
+$block_class     = $attributes['blockClass'] ?? '';
+$style_align     = isset( $paragraph['styleAlign'] ) ? "{$component_class}__align--{$paragraph['styleAlign']}" : '';
+$style_color     = isset( $paragraph['styleColor'] ) ? "{$component_class}__color--{$paragraph['styleColor']}" : '';
+$style_size      = isset( $paragraph['styleSize'] ) ? "{$component_class}__size--{$paragraph['styleSize']}" : '';
+
+$paragraph_class = "
+  {$component_class}
+  {$style_color}
+  {$style_align}
+  {$style_size}
+  {$block_class}__paragraph
+";
```

Comment:
We have the equivalent `classNames()` function (from JS) in PHP, you can do it like:

```php
<?php

Components::classnames([
  $component_class,
  $style_color,
  $style_align,
  $style_size,
  "{$block_class}__paragraph"
]);
```

Reply:
Tnx